### PR TITLE
1705 BE Katie Shermer

### PR DESCRIPTION
**Pull request by katiekeel**

### CODE REVIEW

**Comment by katiekeel** [comment]
Wow, I just realized I *completely forgot* to take the Pizza module into account. But my tests do pass...

File: lib/customer.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  end
+
+  def order_in_place?
+    @order_in_place
+  end
+
+  def place_order(pizza, date_time)
+    @order_in_place = true
+    @orders << [pizza, date_time]
+    @order_frequency += 1
```
Comment (mikedao) [inline lib/customer.rb:34]: Is there a better way to calculate this?

File: lib/customer.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    @orders.each do |order|
+      if (DateTime.now.to_date - order[1].to_date) > 30
+        @order_frequency -= 1
+      end
+    end
+  end
+
+  def frequent_customer?
+    calculate_order_frequency
+    true if @order_frequency > 2
```
Comment (mikedao) [inline lib/customer.rb:47]: I might put this in another method.

File: lib/pizza.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    { :size => @size,
+      :type => @type,
+      :crust => @crust }
+  end
+
+  def calculate_price
+    if @size == "medium"
+      12.50
+    elsif @size == "large"
+      15.00
```
Comment (mikedao) [inline lib/pizza.rb:21]: Look at your prices.rb

**Pull request closed without merging**